admin/class-accessible-table.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Links
+ */
+
+/**
+ * Represents the state of a table being accessible.
+ */
+abstract class WPSEO_Accessible_Table {
+
+	const ACCESSIBLE   = '0';
+	const INACCESSBILE = '1';
+
+	/**
+	 * The storage class used to check the accessible table.
+	 *
+	 * @var string
+	 */
+	public static $storage_class = 'WPSEO_Link_Storage';
+
+	/**
+	 * The transient used to store the accessible state.
+	 *
+	 * @var string
+	 */
+	public static $transient_name = 'wpseo_link_table_inaccessible';
```

Comment:
I would set an empty string as default value, because this is an implementation detail of the class.